whist/core/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Retrieves envs for the whist core module.
+"""
+import os
+
+ALGORITHM = os.getenv('ALGORITHM', 'HS256')
+SECRET_KEY = os.getenv('SECRET_KEY', 'geheim')
```

Comment:
any reason why to use this algorithm?
just curious

Reply:
Copied from documentation: https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/